src/rubrix/server/apis/v0/models/metrics/token_classification.py

```diff
@@ -417,22 +419,26 @@ def build_tokens_metrics(
                 idx=token_idx,
                 value=token_value,
                 char_start=char_start,
-                char_end=char_end,
+                char_end=char_end - 1,
```

Comment:
I think the `char_end` definition should be changed to `text[char_start:char_end] == "token"`. Right now it is `text[char_start:char_end+1] == "token"`, that is why I added a `- 1`.

Reply:
The `char_end` character should be aligned to the way annotations are defined, so, text[char_start:char_end] == "token", but we will tackle this in a separate PR.